### PR TITLE
feat: add contribute guideline

### LINKED ISSUE #7: 开发指南文档细化
https://github.com/bestchains/website/tree/main/docs/DevGuide

### PR DESCRIPTION
**Pull request by Abirdcfly**
Fix #7

### CODE REVIEW

**Comment by Abirdcfly** [comment]
see https://abirdcfly.github.io/website/docs/Contribute/guidelines for preview

**Comment by bjwswang** [comment]
pls add english document too 
https://github.com/bestchains/website/tree/main/i18n/en/docusaurus-plugin-content-docs

File: docs/Contribute/guidelines.md [hunk 1]
```diff
@@ -1,7 +1,131 @@
...
 
+我们鼓励您通过报告问题、改进文档、修复错误或添加新功能来为社区做贡献!
 
+目前我们的行为准则遵循 CNCF 的[行为准则](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)，其中详细说明了您作为 Bestchains 社区的一员应如何行事。
+
+## 项目组成
+
+Bestchains 由若干个项目组成：
+
+- [fabric-operator](https://github.com/bestchains/fabric-operator): 用于 k8s 环境中的 fabric operator
```
Comment (bjwswang) [inline docs/Contribute/guidelines.md:16]: 基于kubernetes operator框架，实现的区块链管理服务。

File: docs/Contribute/guidelines.md [hunk 2]
```diff
@@ -1,7 +1,131 @@
...
+- [bc-console](https://github.com/bestchains/bc-console): Bestchains 相关的 console
```
Comment (bjwswang) [inline docs/Contribute/guidelines.md:17]: `bc-console` and `bc-apis` 放在一起比较合适

File: docs/Contribute/guidelines.md [hunk 3]
```diff
@@ -1,7 +1,131 @@
...
+- [installer](https://github.com/bestchains/installer): Bestchains 环境相关安装文件
```
Comment (bjwswang) [inline docs/Contribute/guidelines.md:18]: 提供Bestchains平台安装的相关Helm charts

File: i18n/en/docusaurus-plugin-content-docs/current/Contribute/guidelines.md [hunk 4]
```diff
@@ -1,7 +1,131 @@
...
+
+We choose GitHub as the primary place for Bestchains collaboration, so the latest updates of Bestchains are always here. Although contributing through PRs is a clear way to help, we still call for any other ways:
+
+- reply to other's issues if you could;
+- help solve other user's problems;
+- help review other's PR design;
+- help review other's codes in PR;
+- discuss about Arbiter to make things clearer;
+- advocate Arbiter technology beyond GitHub;
+- write blogs on Arbiter and so on.
```
Comment (0xff-dev) [inline i18n/en/docusaurus-plugin-content-docs/current/Contribute/guidelines.md:107]: Arbiter -> Bestchains.

**Review by bjwswang — APPROVED** [review]
/lgtm

**Pull request merged**